llvm/test/Transforms/ExpandLargeFpConvert/X86/expand-large-fp-optnone.ll

```diff
@@ -0,0 +1,252 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=x86_64 < %s | FileCheck %s
```

Comment:
Codegen tests do not belong in test/Transforms. this should just run opt with the pass

Reply:
I changed it.